api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -128,6 +128,18 @@ private OpenmrsUtil() {
 	
 	private static Map<Locale, SimpleDateFormat> timeFormatCache = new HashMap<>();
 	
+	public final static String tagStart = "\\<\\w+((\\s+\\w+(\\s*\\=\\s*(?:\".*?\"|'.*?'|[^'\"\\>\\s]+))?)+\\s*|\\s*)\\>";
```

Comment:
Which regex expression is this, do you mind helping me to understand it

Reply:
Hi Team,
I have already implemented an alternate fix that has been merged by @isears for this issue. Reviewers, Kindly cancel this PR.

Reply:
can you share the pull request here please thanks @parth59